samples/kubernetes/end2end/docs/03-wls-operator.md

```diff
@@ -1,41 +1,7 @@
 ## Installing the WebLogic Server Kubernetes Operator
 We depend on the [WebLogic Server Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator) to create and manage WebLogic domains. For detailed installation information, see [Install the Operator](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/installation/).
```

Comment:
https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-operators/installation/

This link has been changed to https://oracle.github.io/weblogic-kubernetes-operator/managing-operators/installation/

Reply:
done